articles/network-types-topologies/index.md

```diff
@@ -0,0 +1,91 @@
+
+![Linear Topology](/engineering-education/network-types-topologies/bus.jpg)
+
+#### Ring Topology
+Ring topology is a special kind of bus topology. *A linear topology in which both ends are joined together forms a **Ring Topology***. Thus no terminators are used and so the data flow is *unidirectional*. Because of that, for the data to reach from the first node to the last  it has to traverse through all the nodes. Since the connection of nodes is continuous, data flows infinitely throughout the topology until one node removes it.
+Like linear topology, the structure and arrangement of nodes are very simple. Due to its compact architecture, the configuration of nodes becomes easier. Since the nodes are connected in a loop, unpredictable troubleshooting problems may appear. To repair or fix one crashed node, the whole system has to be taken down. It is difficult to shut down one node and keep the rest of the system functional. Like the linear topology, adding new devices and components may slow down the data transfer rate.
+
+![Ring Topology](/engineering-education/network-types-topologies/ring.jpg)
+
+#### Mesh Topology
+Mesh topology is the solution to the problem with ring topology. Unlike ring topology, here *every node is connected with every other node present in the network*. In the case of a single node failure, the rest of the system can still be functional. Since one node connects many other nodes, data is shared through any one of the many paths available. This makes mesh arrangement a suitable network for industries and business establishments. A mesh topology in which all the nodes are connected to each other is called full connectivity, and the mesh topology in which only some nodes are connected to each other is called partial connectivity.
+Mesh topology increases the network flow and data can be shared faster. It also records a lesser number of network breakages and system breakdowns. But since all the nodes are connected to each other, each node low key acts as a router to another node and so on. This increases power consumption to a larger extent. Mesh topology needs lots of proper planning and requires high maintenance. Due to many links, high network latency may be an issue.
+
+![Mesh Topology](/engineering-education/network-types-topologies/mesh.jpg)
+
+#### Star Topology
+*This topology consists of an extra part, i.e. hub or switch, in the center of the topology which connects all the nodes*. A hub or switch is a networking device that transfers data from one device to another device. It acts as a transmission medium. While both are used depending on their operations, some [differences](https://www.geeksforgeeks.org/difference-between-hub-and-switch/) may affect the way they perform. This is the most commonly used topology.

```

Comment:
"This is the most commonly used topology." 
Maybe a link supporting this claim and/or an explanation as to why its the most common - what makes it the ideal choice? 🚀

Reply:
Restructured the paragraph(A/D).
Added another point as to why star topology is the most used one.